harvesters/src/main/java/org/fao/geonet/kernel/harvest/harvester/csw/Aligner.java

```diff
@@ -150,12 +157,12 @@ public Aligner(AtomicBoolean cancelMonitor, ServiceContext sc, CswServer server,
 
     }
 
-    public HarvestResult align(Set<RecordInfo> records, List<HarvestError> errors) throws Exception {
+    public HarvestResult align(Collection<RecordInfo> records, Collection<HarvestError> errors) throws Exception {
         if (cancelMonitor.get()) {
             return result;
         }
 
-        log.info("Start of alignment for : " + params.getName());
+        log.debug("Start of alignment for : " + params.getName());
```

Comment:
About the changes from `info` to  `debug` have you check the execution log files of the harvesters that are not forcing explicitly the `info` level? I remember time ago was changed so the log files provided `info` level messages and users could get some useful log of the execution, as previously the only way was to use `debug` level, but was too much verbose.

Reply:
As far as I know, the log level is the one defined on log4j.xml. Unless there was some development I am not aware of. Going to check.